docs/copied-from-beats/docs/security/users.asciidoc

```diff
@@ -197,156 +344,245 @@ information.
+|`Read` on {beat_kib_app}
+|Allow the use of the {beat_kib_app}
+|====
 
-To grant users the required privileges:
+[[privileges-agent-central-config-kib]]
+==== Kibana central configuration management
 
-ifndef::apm-server[]
-. Create a *reader role*, called something like +{beat_default_index_prefix}_reader+, that has
-the following privilege:
+. If you're using APM Agent configuration, create a user with the following built-in roles:
++
+[options="header"]
+|====
+|Role | Purpose
+
+|`apm_user`
+|Read and update APM Agent configuration via {kib}
+|====
++
+. The user will also need Kibana space access:
 +
 [options="header"]
 |====
 |Type | Privilege | Purpose
 
-|Index
-|`read` on +{beat_default_index_prefix}-*+ indices
-|Read data indexed by {beatname_uc}
+| Spaces
+|`All` on {beat_kib_app}
+|Allow the use of the {beat_kib_app}
 |====
 
-. Assign the *reader role*, along with the following built-in roles, to
-users who need to read {beatname_uc} data:
+TIP: Looking for privileges and roles needed for the central configuration
+API? See <<privileges-agent-central-config-api>>.
+
+////
+***********************************  ***********************************
+***********************************  ***********************************
+////
+
+// [[privileges-create-api-keys]]
+// === Grant privileges and roles needed to create APM Server API keys
+
+// ++++
+// <titleabbrev>Create an _APM API key_ user</titleabbrev>
+// ++++
+
+// CONTENT
+
+////
+***********************************  ***********************************
+***********************************  ***********************************
+////
+
+[[privileges-apm-app-api]]
+=== Grant privileges and roles needed for APM app APIs
+
+++++
+<titleabbrev>Create an _APM app API_ user</titleabbrev>
+++++
+
+[[privileges-apm-app-annotation-api]]
+==== Annotation API
+
+. Create a user with the following privileges:
 +
 [options="header"]
 |====
-|Role | Purpose
+|Type | Privilege | Purpose
 
-|`kibana_user` or `kibana_dashboard_only_user`
-|Use {kib}. `kibana_dashboard_only_user` grants read-only access to dashboards.
+|Index
+|`create_doc` on +observability-annotations+ indices
+|Write events into {es}
+
+|Index
+|`create_index` on +observability-annotations+ indices
+|Create the indices
 
-ifdef::has_central_config[]
-|`beats_admin`
-|Create and manage configurations in Beats central management. Only assign this
-role to users who need to use Beats central management.
-endif::[]
+|Index
+|`manage` on +observability-annotations+ indices
+|Manage the indices
+
+|Index
+|`read` on +observability-annotations+ indices
+|Read the indices
 |====
 +
-Omit any roles that aren't relevant in your environment.
-endif::apm-server[]
+. The user will also need Kibana space access:
++
+[options="header"]
+|====
+|Type | Privilege | Purpose
+
+| Spaces
+|`All` on {beat_kib_app}
+|Allow the use of the {beat_kib_app}
+|====
+
+[[privileges-agent-central-config-api]]
+==== Central configuration management API
```

Comment:
Why is this a dedicated section and not part of the https://github.com/elastic/apm-server/pull/3799/files#diff-d57f94e88b4f7a29666f5fd313851fbfR474 section?

Reply:
Because I'm planning to move the APM app content to the APM app docs but was just staging everything here to get it correct and see it all in one place :)